entrypoint.sh

```diff
@@ -109,8 +109,19 @@ function appsodyStart() {
 		cmd=debug
 	fi
 
-	if [ -f "env.properties" ]; then
-		dopts=--docker-options="--env-file=env.properties"
+	ENV_PROPERTIES_FILE=env.properties
+	PROJECT_LINKS_ENV_FILE="$ROOT/.codewind-project-links.env"
```

Comment:
This file is at the root of the project, just like `env.properties`?  If so then `$ROOT/` part is not needed, the root is already working directtory.  It doesn't hurt to prefix it tho, but let's make it consistent so either both prefixed or both not.